examples/lusid/orders/blocks/2-call-block.sql

```diff
@@ -0,0 +1,20 @@
+/*
+
+-----------
+call Block
+-----------
+
+Description:
+
+    - In this query, we will call the Blocks that i just set
```

Comment:
"retrieve the Block that has been loaded"

Reply:
updated